app/vep/models/pipeline_model.py

```diff
@@ -60,12 +74,12 @@ def create_config_ini_file(self):
 biotype {biotype}
 transcript_version {self.transcript_version}
 canonical {self.canonical}
-'''
+"""
 
-    try:
-      with tempfile.NamedTemporaryFile(prefix="vep_", suffix=".ini", dir=UPLOAD_DIRECTORY, delete=False) as ini_file:
-        ini_file.write(config_yaml.encode())
-      return ini_file
-    except Exception as e:
-      logging.info(e)
-      raise RuntimeError("Could not create vep config ini file")
+        try:
+            with open(os.path.join(dir, "config.ini"), "w") as ini_file:
```

Comment:
Out of curiosity and ignorance: is this any better than the tempfile approach?

Reply:
Tempfile would just add complexity with no benefit over static filename (random filename, need to handle cleanup, unneeded operations).

Reply:
I think initial assumption was to store in a UPLOAD_DIRECTORY but it make sense to have it in the same directory as vcf data since the directory is unique.